x/claim/types/msgs.go

```diff
@@ -0,0 +1,47 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// bank message types
+const (
+	TypeMsgClaim = "send"
```

Comment:
Why is this the type string?

Reply:
Msg Types are string

Reply:
No sorry, I meant shouldn't this be "claim" instead of "send"?

Reply:
I see, but it's removed as you recommended on other comment. All are done on hook including withdraw.